containers/podman-desktop
# feat: add compose extension

### What does this PR do?
extension is responsible to download docker-compose and setup a `podman-desktop` binary that will setup properly DOCKER_HOST

### Screenshot/screencast of this PR

https://user-images.githubusercontent.com/436777/221840559-5d14c54e-22e6-4546-8fd4-a6f6f6c10154.mp4

### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/1478

### How to test this PR?

Start and click on compose in status bar, it should guide you to setup `podman-compose` binary


Change-Id: Id0e6628ca5860649c2a0b19f7a85eb326605c9ac
